workflowai/core/domain/run.py

```diff
@@ -86,6 +87,47 @@ def model(self):
             return None
         return self.version.properties.model
 
+    def format_output(self) -> str:
+        """Format the run output as a string.
+
+        Returns a formatted string containing:
+        1. The output as a nicely formatted JSON object
+        2. The cost with $ prefix (if available)
+        3. The latency with 2 decimal places and 's' suffix (if available)
+
+        Example:
+            Output:
+            ==================================================
+            {
+              "message": "hello"
+            }
+            ==================================================
+            Cost: $ 0.001
+            Latency: 1.23s
+        """
+        # Create a dictionary with the output data
+        output_data = self.output.model_dump()
+
+        # Format the output string
+        output = [
+            "\nOutput:",
+            "=" * 50,
+            json.dumps(output_data, indent=2),
+            "=" * 50,
+        ]
+
+        # Add run information if available
+        if self.cost_usd is not None:
+            output.append(f"Cost: $ {self.cost_usd}")
+        if self.duration_seconds is not None:
+            output.append(f"Latency: {self.duration_seconds:.2f}s")
+
+        return "\n".join(output)
+
+    def print_output(self) -> None:
+        """Print the output in a nicely formatted way."""
+        print(self.format_output())
```

Comment:
```suggestion
        print(self.format_output()) # noqa: T201
```

printing is usually bad in production code but ok here.

Another way to implement this functionality would be to rename `format_output` to `__repr__` (or `__str__` if `__repr__` does not work)

And this way you would get pretty printing when calling `print(run)` and not need `print_output`

Reply:
OK I agree `__str__` is way cleaner!